### PR TITLE
[CCUBE-1116][AC] add customOptions for indicator and bump design system version

### PR DESCRIPTION
**Pull request by alvinchow97**
[CCUBE-1116][AC] add customOptions for indicator and bump design system version

**Changes**
1. Add showIndicatorLabel, indicatorLabelPrefix, indicatorLabelSuffix

-   [delete] branch

-   You may refer to this [[ticket](url)](https://jira.ship.gov.sg/browse/CCUBE-1116)

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -55,7 +55,7 @@
 		"yup": "^0.32.11"
 	},
 	"peerDependencies": {
-		"@lifesg/react-design-system": "^2.3.0-canary.7",
+		"@lifesg/react-design-system": "^2.4.0-canary.4",
```
Comment (qroll) [inline package.json:58]: can trouble you to reflect the latest version in the `README.md` too?

File: src/stories/3-fields/slider/slider.stories.tsx
Comment (qroll) [inline src/stories/3-fields/slider/slider.stories.tsx:1]: can update the `Labels` story to show an example of usage with the following options

```
showIndicatorLabel: true,
indicatorLabelPrefix: "Within $",
indicatorLabelSuffix: ".00",
```

File: src/stories/3-fields/slider/slider.stories.tsx [hunk 2]
```diff
@@ -41,10 +41,10 @@ const meta: Meta = {
 		},
 		customOptions: {
 			description:
-				"<ul><li>`showSliderLabels` specifies if max and min labels are displayed.</li><br/><li>Use `sliderLabelPrefix` and `sliderLabelSuffix` to customise the labels</li></ul>",
+				"<ul><li>`showSliderLabels` specifies if max and min labels are displayed.</li><br/><li>Use `sliderLabelPrefix` and `sliderLabelSuffix` to customise the labels</li><br /> <li> `showIndicatorLabel` specifies if indicator will be displayed </li> <br/> <li> Use `indicatorLabelPrefix` and `indicatorLabelSuffix` to customise the the indicator. </li> <br/> </ul>",
```
Comment (qroll) [inline src/stories/3-fields/slider/slider.stories.tsx:44]: small typo, extra `the` set for the last line

```suggestion
				"<ul><li>`showSliderLabels` specifies if max and min labels are displayed.</li><br/><li>Use `sliderLabelPrefix` and `sliderLabelSuffix` to customise the labels</li><br /> <li> `showIndicatorLabel` specifies if indicator will be displayed </li> <br/> <li> Use `indicatorLabelPrefix` and `indicatorLabelSuffix` to customise the indicator. </li> <br/> </ul>",
```

**Review by qroll — CHANGES_REQUESTED** [review]

File: package.json (hunk 1 above)
Comment (alvinchow97) [reply package.json:58]: Updated, thanks for remind

File: src/stories/3-fields/slider/slider.stories.tsx (hunk 2 above)
Comment (alvinchow97) [reply src/stories/3-fields/slider/slider.stories.tsx:44]: Noted, removed the extra the

File: src/stories/3-fields/slider/slider.stories.tsx
Comment (alvinchow97) [reply src/stories/3-fields/slider/slider.stories.tsx:1]: Noted, added to the Labels story

**Review by weili-govtech — APPROVED** [review]

**Review by qroll — APPROVED** [review]

**Pull request merged**